apps/full-site-editing/package.json

```diff
@@ -96,7 +96,7 @@
 		"@wordpress/compose": "*",
 		"@wordpress/data": "*",
 		"@wordpress/date": "*",
-		"@wordpress/dependency-extraction-webpack-plugin": "^2.6.0",
+		"@wordpress/dependency-extraction-webpack-plugin": "^2.7.0",
```

Comment:
note to self: we should probably run the FSE plugin GH actions when the packages are updated. cc @vindl

Reply:
we even rely on changes in the root package.json

Reply:
the FSE plugin workflow will run on changes to `yarn.lock` in https://github.com/Automattic/wp-calypso/pull/43458

Reply:
We reverted that behavior in #43480 since the Action was running too often (on every Renovate PR, even those that didn't affect FSE).